README.md

```diff
@@ -37,6 +37,9 @@ It has been made clear that the project can be deployed to Jetty using Maven. Bu
 kb-api.xml file must be corrected to point to logback and properties files where they are located on the server. 
 For local tomcat installation, you can deploy kb-api.xml via symlink to tomcat/conf/Catalina/localhost <br>
 
+The project is configured to run Jetty locally as well (local installation is not required). <br>
+It is important to use jetty: run-war when starting jetty server. It wraps the application in a war file first and deploys to Jetty server afterwards. Jetty is watching the pom.xml file and re-deploying the application as soon as there are changes.
```

Comment:
Mindre formatteringsnote: Det er bedst at skrive
```
...to use `jetty:run-war` when...
```
så man som læser af README får det vist som "...to use `jetty:run-war` when...". Det gør det nemmere at skimme gennem teksten efter hvad der skal gøres.